src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1011.ReferenceFinalSchema_Invalid.sarif

```diff
@@ -52,6 +52,14 @@
               "descriptor": {
                 "id": "WRN999.RuleExplicitlyDisabled"
               }
+            },
+            {
+              "message": {
+                "text": "Rule 'SARIF2016' was explicitly disabled by the user. As result, this tool run cannot be used for compliance or other auditing processes that require a comprehensive analysis."
```

Comment:
>SARIF2016 [](start = 31, length = 9)

Here's a clue we're off track: our current approach requires us to emit an explicit change to a large set of test files to accommodate a new rule. Changing these rules to 'note' and requiring an explicit opt-in argument to see 'note' messages will eliminate this requirement.